assets/src/hooks/usePanelState.ts

```diff
@@ -53,13 +53,16 @@ export const usePanelStateForViewState = (
         })
       }
     },
-    openVehiclePropertiesPanel: (
-      vehicle: VehicleType,
-      _initialView?: TabMode
-    ) =>
+    setTabMode(tabMode: TabMode) {
```

Comment:
**nitpick:** is there a reason to use the different notation without the `=>` for this function?

Reply:
Oh y'know, I ment for all of them to follow this convention, instead of the arrow function convention, but given the rest are using arrow functions, I'll update this to match.

Reply:
Squashed into commit 13d46768878283a58025d4054f9e40d0f4197520

Reply:
I actually don't have a strong preference between the two as long as it's consistent, though I think the arrow notation is more common in our codebase overall.